.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,44 @@
+name: Bug Report🐛
+description: Create a report to help us improve
+title: "Bug🐛: "
+labels: [bug]
+
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for taking the time to file a Bug report
+  - type: textarea
+    attributes:
+      label: Describe the bug.
+      description: "Enter a clear and concise description of what the bug is"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Describe the steps to reproduce the behavior
+      description: "Be clear and concise while describing the steps to ensure that it can be reproduced and potentially fixed"
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+  - type: textarea
+    attributes:
+      label: Expected behavior.
+      description: "Enter a clear and concise description of what you expected to happen"
+  - type: textarea
+    attributes:
+      label: Screenshots.
+      description: "If applicable, add screenshots to help explain your problem"
+  - type: input
+    attributes:
+      label: Operating environment
+      description: "What operating system and version are you using? If you are running it on Cloud/Cluster do mention about that."
```

Comment:
Perhaps more concisely: "Are you running on a cloud/cluster?"

Reply:
I change whole template so new comers and everyone can understand very well